students/BrianNisonger/Session8/test_Circle.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+from circle import circle
+from circle import sphere
+import math
+
+
+def test_Circle():
+    c = circle(10)
+    assert c.radius == 10
+
+
+def test_diameter():
+    c = circle(10)
+    assert c.diameter == 20
+
+
+def test_set_diameter():
+    c = circle(10)
+    assert c.diameter == 20
+    c.diameter = 10
+    assert c.diameter == 10
+
+
+def test_change_radius_diameter():
+    c = circle(10)
+    c.diameter = 100
+    assert c.radius == 50
+
+
+def test_set_area():
+    c = circle(10)
+    assert math.isclose(c.area, (math.pi * 10**2), rel_tol=.01)
```

Comment:
nice!

though a relative tolerance of 0.01 is really huge! essentially only 2 significant figures -- this should easily support the default (which I think is 1e-8) and probably much more: try 1e-14 or so.